packages/wallet/primitives/src/permission.ts

```diff
@@ -25,7 +25,7 @@ export type SessionPermissions = {
   chainId: number
   valueLimit: bigint
   deadline: bigint // uint64
-  permissions: [Permission, ...Permission[]]
```

Comment:
This typing [Permission, ...Permission[]] is cute typing trick that uses a tuple type to specify that there must be at least 1 permission given in the array. where as Permission[] allows an empty array. Was this change intended?